modules/client/test/integration/client-addressList/estimation.test.ts

```diff
@@ -62,7 +62,7 @@ describe("Client Address List", () => {
 
       const voteParams: IVoteProposalParams = {
         pluginAddress: "0x1234567890123456789012345678901234567890",
-        proposalId: "0x1234567890123456789012345678901234567890",
+        proposalId: "0x0",
```

Comment:
Why 0x0?

Reply:
this should be a `BigNumberish`, we can lock it to a `bigint` and solve the other issues with the `BigNumber` validator